frontend/src/lib/utils/kea-logic-builders.ts

```diff
@@ -0,0 +1,15 @@
+import { BuiltLogic, afterMount } from 'kea'
+/**
+ * Some kea logics are used heavily across multiple areas so we keep it mounted once loaded with this trick.
+ */
+export function permanentlyMount(afterMountFn?: (logic: BuiltLogic) => void): (logic: BuiltLogic) => void {
+    return (logic) => {
+        afterMount(() => {
```

Comment:
I think it'd be slightly clearer the way Marius proposed (keeping `afterMount()` and `permanentlyMount` separate), but that's very minor

Reply:
I was mostly thinking of reducing boilerplate but happy to change it